airbyte-migration/src/main/java/io/airbyte/migrate/Migrations.java

```diff
@@ -56,7 +57,8 @@
   private static final Migration MIGRATION_V_0_25_0 = new MigrationV0_25_0(MIGRATION_V_0_24_0);
   private static final Migration MIGRATION_V_0_26_0 = new MigrationV0_26_0(MIGRATION_V_0_25_0);
   private static final Migration MIGRATION_V_0_27_0 = new MigrationV0_27_0(MIGRATION_V_0_26_0);
-  private static final Migration MIGRATION_V_0_28_0 = new MigrationV0_28_0(MIGRATION_V_0_27_0);
+  public static final Migration MIGRATION_V_0_28_0 = new MigrationV0_28_0(MIGRATION_V_0_27_0);
```

Comment:
Making the variable public is probably better than having to do the following to locate a migration in the unit test:

```
final MigrationV0_28_0 migration = (MigrationV0_28_0) Migrations.MIGRATIONS
    .stream()
    .filter(m -> m instanceof MigrationV0_28_0)
    .findAny()
    .orElse(null);
assertNotNull(migration);
```

I think we should do this from the beginning.

Reply:
thanks! I agree.